### PR TITLE
Fix/tiff in-source build

### PR DESCRIPTION
**Pull request by ResearchDaniel**
Fixes 
* Hide tiff release project under ext/tiff

Changes proposed in this PR:
* Allow building in the source directory
 * Ignore folders build and out (Visual Studio uses out per default)

This has been tested on: Windows, Visual studio

### CODE REVIEW

File: ext/tiff/CMakeLists.txt [hunk 1]
```diff
@@ -207,7 +207,8 @@ message(STATUS "")
 #[===[ Inviwo modification ]===]
 ivw_suppress_compiler_warnings(tiff)
 ivw_default_install_targets(tiff)
-ivw_folder(tiff ext)
+ivw_folder(tiff ext/tiff)
+ivw_folder(release ext/tiff)
```
Comment (petersteneteg) [inline ext/tiff/CMakeLists.txt]: better to remove the include(Release) higher up instead...

**Review by petersteneteg — DISMISSED** [review]

File: ext/tiff/CMakeLists.txt (hunk 1 above)
Comment (ResearchDaniel) [reply ext/tiff/CMakeLists.txt]: Yes and no. The release writes the version in the header. Not sure if any code relies on that

Comment (petersteneteg) [reply ext/tiff/CMakeLists.txt]: Should not be needed there is already a generated file in the in the source tree. So Im fairly sure we will not need this.

Comment (petersteneteg) [reply ext/tiff/CMakeLists.txt]: Also I really dislike having a generic target name like that around if it not inviwo related...

**Review by petersteneteg — DISMISSED** [review]

**Comment by petersteneteg** [comment]
I was expecting that doing a rebase and force push, would discard my review, but it seems it was smart enough not to do that somehow, nice!

**Comment by ResearchDaniel** [comment]
> I was expecting that doing a rebase and force push, would discard my review, but it seems it was smart enough not to do that somehow, nice!

Cool, but I added .vs folder to ignore so you have to review again anyway :D

**Review by petersteneteg — APPROVED** [review]

**Pull request merged**